controllers/gatekeeper_controller.go

```diff
@@ -1038,7 +1039,31 @@ func openShiftDeploymentOverrides(obj *unstructured.Unstructured) error {
 
 	err = unstructured.SetNestedField(obj.Object, containers, "spec", "template", "spec", "containers")
 	if err != nil {
-		return errors.Wrapf(err, "Failed to set the OpenShift overrides")
+		return errors.Wrapf(err, "Failed to set --disable-cert-rotation for OpenShift")
```

Comment:
This is setting all the overrides including securityContexts, not just disable cert rotation.
```suggestion
		return errors.Wrapf(err, "Failed to set the OpenShift overrides for the Deployment containers")
```